index.md

```diff
@@ -10,9 +15,13 @@ You can find compiler status reports over on: [https://github.com/Rust-GCC/Repor
 
 ## Status reports
 
-* [2022 Yearly report](/reporting/2022-year-report.md)
-* [November 2022 monthly report](/reporting/2022-11-monthly-report.md)
-* [October 2022 monthly report](/reporting/2022-10-monthly-report.md)
+<ul>
+  {% for post in site.posts %}
+    <li>
+      <a href="{{ post.url }}">{{ post.title }}</a>
+    </li>
+  {% endfor %}
+</ul>
```

Comment:
Mh, I wonder: how are the posts being sorted? Alphabetically I assume? In that case, we need to use titles such as `2022 11 monthly report` instead of `November 2022 monthly report`

Reply:
By default, it should be sorted by date (from the metadata in each post).

Reply:
should? :D

Also, will they be reverted? We want the most recent post to be at the top of the page IMO

Reply:
I could not find any confirmation about how it's sorted when used like this, so I made it explicit.

Here is how it's rendered : 

![image](https://user-images.githubusercontent.com/30369999/212758285-334918e4-9172-4b5d-9b1f-e908aa337051.png)